canonical/ubuntu.com
# WD-7763 - redesign /managed page

## Done

Redesign [managed services page](https://ubuntu-com-13445.demos.haus/managed) according to [figma](https://www.figma.com/file/Qj0DByXnYYUn8BMtbB2OiA/24.04-Managed?type=design&node-id=0-1&mode=design&t=vHKZ7F3He1R87Ah5-0) and [copy doc](https://docs.google.com/document/d/19GoMudyrd9Zt1e7kfj-koL-Lt0RGYzWHRPuuKYUPvqw/edit)

## QA

- [demo link](https://ubuntu-com-13445.demos.haus/managed)
- [copy doc](https://docs.google.com/document/d/19GoMudyrd9Zt1e7kfj-koL-Lt0RGYzWHRPuuKYUPvqw/edit)
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Run through the following [QA steps](https://discourse.canonical.com/t/qa-steps/152)
- Check that the page is correct according to the design and copy doc

## Issue / Card
[WD-7763](https://warthogs.atlassian.net/browse/WD-7763)

Fixes #

## Screenshots


[WD-7763]: https://warthogs.atlassian.net/browse/WD-7763?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
